platform/ui/src/components/simpleDialog/SimpleDialog.js

```diff
@@ -47,6 +48,144 @@ class SimpleDialog extends Component {
     );
   };
 
+  static AnnotationMAMADialog = ({ onClose, onSubmit }) => {
```

Comment:
These should probably have their own component files and leverage the UIDialogService:

https://docs.ohif.org/services/ui/ui-dialog-service.html